src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -330,6 +341,41 @@ export default {
 			}
 		},
 
+		/**
+		 * WebImage handlers
+		 */
+		async selectSocialAvatar() {
+
+			if (!this.loading) {
+
+				this.loading = true
+				try {
+					const response = await axios.get(generateUrl('/apps/contacts/api/v1/social/avatar/{id}/{uid}', {
+						id: this.contact.addressbook.id,
+						uid: this.contact.uid,
+					}))
+					if (response.status !== 200) {
+						throw new URIError('verify social profile id')
```

Comment:
That's not really true :)
There are a few options where it can fail and be true:
- The user disabled the facebook graph api
- Network problem or the domain is blacklisted in the server's infrastructure
- ... something else?

We should make sure to tell the user to double check the uid only if we're sure it's true :)
So depending on the result of the avatar fetch. But don't say too much, the user don't care to know if the request failed because of connectivity issue of course. I would only warn if the uid is invalid and we cannot fetch the avatar :thinking:

Reply:
True. will change the error message into a more generic: Download of social profile avatar failed.
Will also add a success-notification, as it may me confusing when the picture hasnt changed in the social network.